### PR TITLE
Query params for filters

### PR DESCRIPTION
**Pull request by barnash**
I made it possible to register filters values in the query string - for dashboard view.
For example:
http://localhost:9001/dashboard/mydashboard?filter1=4&filter2=5

### CODE REVIEW

File: rd_ui/app/scripts/controllers/dashboard.js [hunk 1]
```diff
@@ -32,22 +32,23 @@
                 // TODO: first object should be a copy, otherwise one of the chart filters behaves different than the others.
                 filters[filter.name] = filter;
                 filters[filter.name].originFilters = [];
+                if (_.has($location.search(), filter.friendlyName)) {
```
Comment (arikfr) [inline rd_ui/app/scripts/controllers/dashboard.js]: How about we use `filter.name`? It has more chances to be more URL friendly (in most cases it's the table column name -> no capital letters and no spaces).

**Comment by barnash** [comment]
@arikfr Yeah, you are right, I thought that it will just be easy to create the url since the friendly name is right there when you want to create he deep link. Ofc name makes much more sense. Changed.

**Comment by arikfr** [comment]
Merged. @barnash congrats on your first contribution :-)

**Comment by arikfr** [comment]
@barnash two things I overlooked:
1. Following this change, dashboard filters are enabled for all dashboards regardless of the feature flag. This is fixed in: 16f0413af8e15153586f55028548b284f400f3c7
2. `filter.name` is in the format of "filter-name::filter", while I thought it's only "filter-name"... This is no so friendly, so I might change it in the future to make it easier.

**Comment by barnash** [comment]
Cool, sounds good.

**Pull request merged**